config/schema/openam_api.schema.yml

```diff
@@ -20,3 +20,84 @@ openam_api.settings:
     openam_api_timeout:
       type: integer
       label: 'Maximum allowed time to connect OpenAM server'
+openam_api_operations.settings:
```

Comment:
@passionatepoorna shouldnt this be:
`openam_api_operations` instead of 'openam_api_operations.settings'

Reply:
Updated.